README.md

```diff
@@ -71,12 +77,12 @@ You can set `DOCKER_ARCH`, `REGISTRY` and `VERSION` in the commandline to build
 ```
 make image DOCKER_ARCH=amd64 REGISTRY=yunikorn VERSION=latest
 ```
-This command will build an amd64 binary executable with version `latest` and the docker image tag is `yunikorn/yunikorn:scheduler-latest`. If not specified, `DOCKER_ARCH` defaults to the build host's architecture.
+This command will build an amd64 binary executable with version `latest` and the docker image tag is `yunikorn/yunikorn:scheduler-amd64-latest`. If not specified, `DOCKER_ARCH` defaults to the build host's architecture.
 
 You can run following command to retrieve the meta info for a docker image build, such as component revisions, date of the build, etc.
 
 ```
-docker inspect --format='{{.Config.Labels}}' yunikorn/yunikorn:scheduler-latest
+docker inspect --format='{{.Config.Labels}}' yunikorn/yunikorn:scheduler-amd64-latest
```

Comment:
This may be a controversial change since you may have developers building accross many different repos.  Not exactly sure how to make this better.  

I could set it to the most common OS for your developers and anyone else can modify this locally.  Open to suggestions on these changes?

Reply:
I think what you have is fine. Maybe we could just put something at the top that wherever "amd64" is present in the examples, it should be replaced with the architecture you are building for.

Reply:
If its fine, I think we can leave it as is.  I added some text to the Makefile to reflect where that tag comes from.  So I think its good now?

Reply:
I think this is ready for review whenever you can get a chance.